rust-toolchain.toml

```diff
@@ -1,4 +1,6 @@
 [toolchain]
-channel = "stable"
+# Release of Rust 1.87.0 introduced issues on CI making tests involving instantiation of wasm smart contracts fail.
+# Pinning version to stable 1.86 until resolved.
+channel = "1.86.0"
```

Comment:
Might be worth using "1.86" so any future patch releases are automatically used.

Reply:
Ah, great call!